components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -47,7 +56,7 @@ public function get_pod_filter( $current_post_type = '', $possible_pods = array(
 		if ( isset( $possible_pods[ $current_post_type ] ) ) {
 			$this_pod = $possible_pods[ $current_post_type ];
 
-			if ( is_singular( $current_post_type ) ) {
+      if ( is_singular( $current_post_type ) ) {
```

Comment:
This spaces indentation is going against code standards.

Reply:
Fixed